.github/workflows/Cypress.yaml

```diff
@@ -0,0 +1,22 @@
+name: Cypress
+
+# Triggers the workflow on push or pull request events but only for the main branch
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      # Install NPM dependencies, cache them correctly
+      # and run all Cypress tests
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          build: npm run build
+          start: npm start
```

Comment:
are you sure this is correct? 

From the docs looks like we have to chain the commands and use it this way https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsrun

Reply:
This is what cypress has: https://docs.cypress.io/guides/continuous-integration/github-actions